.github/workflows/build-gui-release-binaries.yml

```
@@ -28,41 +28,62 @@ jobs:
-            libappindicator3-dev \
-            librsvg2-dev;
-
-          sudo apt install -y \
-            libwebkit2gtk-4.1-0=2.44.0-2 \
-            libwebkit2gtk-4.1-dev=2.44.0-2 \
-            libjavascriptcoregtk-4.1-0=2.44.0-2 \
-            libjavascriptcoregtk-4.1-dev=2.44.0-2 \
-            gir1.2-javascriptcoregtk-4.1=2.44.0-2 \
-            gir1.2-webkit2-4.1=2.44.0-2;
+          sudo apt install -y ${{ env.DEPS_MONERO_LINUX }} ${{ env.DEPS_GUI_UBUNTU_SPECIFIC }} git
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)